pages/about.html

```diff
@@ -106,8 +106,5 @@ <h1 class="underline-center partners-title">Partners</h1>
         </div>
     </footer>
 </body>
-<script src="../scripts/staticData.js"></script>
```

Comment:
- [ ] Kindly link your About page properly
<img width="1384" alt="Screenshot 2023-05-31 at 05 21 23" src="https://github.com/habitoz/serategna/assets/81444338/d76bc959-06e9-4e9b-b07d-a17cb4ac027e">

Reply:
> * [ ]  Kindly link your About page properly
> 
> <img alt="Screenshot 2023-05-31 at 05 21 23" width="1384" src="https://user-images.githubusercontent.com/81444338/242272949-d76bc959-06e9-4e9b-b07d-a17cb4ac027e.png">

As requested by the previous reviewer, kindly link your about page properly in order to avoid the 404 message.